docs/componentExtension.md

```diff
@@ -0,0 +1,83 @@
+## Customizing and Extending Components
```

Comment:
We should have an introduction here explaining what needs to be done: 1- extend the data model 2- extend the query for the client component. These are 2 distinct steps.

Reply:
@priandsf thank you! I will add.